manifests/d/DashoGames/PiozaGameLauncher/0.4.0/DashoGames.PiozaGameLauncher.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: DashoGames.PiozaGameLauncher
+PackageVersion: 0.4.0
+PackageLocale: en-US
+Publisher: DashoGames
+PackageName: Pioza Game Launcher

```

```suggestion
PackageName: PiozaGameLauncher
```
This should match Control Panel